### DISCUSSION TITLE
Remapping without `omap`?

### DISCUSSION THREAD

**Discussion by Acumane** [esm7/obsidian-vimrc-support#174] [Q&A]
Say I've remapped `hjkl` to `ijkl` (to be like `wasd` or arrow keys) and let `h` now enter insert mode:

``` 
noremap i gk
noremap j h
noremap k gj
noremap h i
```

I'm not using `map` or binding to, say, `<Up>` in the above snippet because they simply wouldn't work (please, prove this wrong!).

Given the circumstances, how can I get motions working without `omap`?
e.g. `dhw`... or even `diw`!

**Comment by userrand** [comment]
I did not test the following but maybe code each case (d, c, y, etc) individually like `nmap yhw yiw`
